machine/machine_core/machine.py

```diff
@@ -244,6 +244,9 @@ def get_admin_group(self):
         else:
             return "wheel"
 
+    def get_cockpit_container(self):
+        return self.execute("podman ps --quiet --all --filter name=cockpit/ws").strip()
```

Comment:
FTR, I originally used ancestor to match on the image name instead of the container name. I was not sure at that time whether we could rely on any particular name, as a simple `podman run <image>` usually creates a non-predictable random name. These days we should use the `RUN` label and a fixed name, so good enough for me if that works.

Conceptually, filtering on `*/cockpit/ws` image name would still be more robust.

Reply:
Note that the name is just `ws`, not `cockpit/ws`.

Reply:
Image name filter does not accept regexp unfortunately - according to the manpage.